### ISSUE TITLE
allow omitting `steps`

### ISSUE THREAD

**Issue by sw-yx** [pomber/code-surfer#26]
hey pomber! love this!!

my bug is simple. this works:

```js
<CodeSurfer
  title="React-Blade API"
  theme={ShadesOfPurple}
  code={require("!raw-loader!./snippets/react-blade.js")}
  lang="javascript"
  showNumbers={false}
  dark={true}
  steps={[]}
/>
```

but this doesnt work

```js
<CodeSurfer
  title="React-Blade API"
  theme={ShadesOfPurple}
  code={require("!raw-loader!./snippets/react-blade.js")}
  lang="javascript"
  showNumbers={false}
  dark={true}
/>
```

the error in browser console is:

```
Uncaught TypeError: Cannot read property 'some' of undefined
    at InnerCodeSurfer.render (deck-code-surfer.js:117)
```

the problematic codesurfer code  is:

```js
      var anyNotes = notes || steps.some(function (s) {
        return s.notes;
      });
```

so basically i suggest checking that steps is truthy before you do `.some`.

**Comment by pomber** [comment]
Hey Shawn, I think Sara fixed that in #25. I'll publish it later today.

**Comment by pomber** [comment]
Published with v0.4.0
